study_html_css/css/mc.css

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.google.com/specimen/Recursive')
+html, body{
+	width: 100%;
+	height: 100%;
+	font-family: recursive !important;
+	font-size: 25px;
+	color: #34465c;
+}

```

Comment:
em có thể chia file css thành nhiều file nhỏ nhưng các file nhỏ đó vẫn tác động lên toàn bộ trang web . thế nên e ko nên style cho html , body những thuộc tính kiểu font-size : 25px . điều này dẫn đến các đoạn text khác trong trang nếu ko định nghĩa lại font-size thì nó sẽ là 25px . mặc định nó là 16px thì phải !